sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,121 @@
-    <item>
-      <EFDB_Name>value</EFDB_Name>
-      <Description>This value is ignored.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processCalibrationProducts</EFDB_Topic>
-    <Description>Process new calibration products.</Description>
+    <EFDB_Topic>MTAOS_command_selectSources</EFDB_Topic>
+    <Description>Run source selection algorithm for a specific field and visit configuration.</Description>
     <item>
-      <EFDB_Name>directoryPath</EFDB_Name>
-      <Description>Absolute directory path to calibration products.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>ra</EFDB_Name>
+      <Description>Right Ascension of the field in ICRS coordinates.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>hourangle</Units>
       <Count>1</Count>
     </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processWavefrontError</EFDB_Topic>
-    <Description>Process the wavefront data collection at corner splitted sensors.</Description>
     <item>
-      <EFDB_Name>visit</EFDB_Name>
-      <Description>Image visit number.</Description>
-      <IDL_Type>int</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>directoryPath</EFDB_Name>
-      <Description>Absolute directory path to wavefront data collection.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>fieldRA</EFDB_Name>
-      <Description>Right ascension of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>decl</EFDB_Name>
+      <Description>Declination of the field in ICRS coordinates.</Description>
+      <IDL_Type>float</IDL_Type>
       <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>fieldDEC</EFDB_Name>
-      <Description>Declination of telescope boresight.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>pa</EFDB_Name>
+      <Description>Sky position angle of the observation. As defined by the IAU, this the angle measured relative to the north celestial pole (NCP), turning positive into the direction of the right ascension.</Description>
+      <IDL_Type>float</IDL_Type>
       <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>filter</EFDB_Name>
-      <Description>Active filter type. Use 1-6 for u, g, r, i, z, and y. Use 7 for reference.</Description>
-      <IDL_Type>int</IDL_Type>
+      <Description>Name of the filter used in the observation.</Description>
+      <IDL_Type>string</IDL_Type>
```

Comment:
I think if the type is `string`, it is better to write down the available options: u, g, r, i, z, y, and ref.

Reply:
Although I agree with the idea of using enums, there are a couple of drawback to doing that. 

Once is that we would be tied to the enums to add support to filters. If eventually we have different filters installed in the camera (something that may happen in the future) we would have to update the enums as well. 

The other "issue" is that, if this is an interface enum, the source of truth must be the xml and not `ts_wep`... that means, we need to add the definition to the xml file and that will have to be added to ts_idl as well.